Build failure: v4l2loopback

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

Build NixOS config including linux-6.15 kernel.

### Can Hydra reproduce this build failure?

Hydra doesn’t try to build the package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
error: build of '/nix/store/0g4mjwl3kg1iwmvjm410xxk9ckcligkn-v4l2loopback-0.13.2-6.15.drv' on 'ssh://root@192.168.178.44' failed: builder for '/nix/store/0g4mjwl3kg1iwmvjm410xxk9ckcligkn-v4l2loopback-0.13.2-6.15.drv' failed with exit code 2
error: builder for '/nix/store/0g4mjwl3kg1iwmvjm410xxk9ckcligkn-v4l2loopback-0.13.2-6.15.drv' failed with exit code 1;
       last 25 log lines:
       > substituteStream() in derivation v4l2loopback-0.13.2-6.15: WARNING: '--replace' is deprecated, use --replace-{fail,warn,quiet}. (file 'Makefile')
       > build flags: SHELL=/nix/store/xy4jjgw87sbgwylm5kn047d9gkbhsr9x-bash-5.2p37/bin/bash ARCH=x86_64 CROSS_COMPILE= KBUILD_OUTPUT=/nix/store/gl4zb4minm4ddaz31lhsx2rr964z1gjl-linux-6.15-dev/lib/modules/6.15.0/build KERNELRELEASE=6.15.0 KERNEL_DIR=/nix/store/gl4zb4minm4ddaz31lhsx2rr964z1gjl-linux-6.15-dev/lib/modules/6.15.0/build
       >
       > ++++++ To sign the  module, you must set KBUILD_SIGN_KEY/KBUILD_SIGN_CERT to point to the signing key/certificate!
       > ++++++ For your convenience, we try to read these variables as 'mok_signing_key' resp. 'mok_certificate' from /etc/dkms/framework.conf
       >
       > ++++++ If your certificate requires a password, pass it via the KBUILD_SIGN_PIN env-var!
       > ++++++ E.g. using 'export KBUILD_SIGN_PIN; read -s -p "Passphrase for signing key : " KBUILD_SIGN_PIN; sudo --preserve-env=KBUILD_SIGN_PIN make sign'
       >
       > Building v4l2-loopback driver...
       > make -C /nix/store/gl4zb4minm4ddaz31lhsx2rr964z1gjl-linux-6.15-dev/lib/modules/6.15.0/build M=/build/source KCPPFLAGS="" modules
       > make[1]: Entering directory '/nix/store/gl4zb4minm4ddaz31lhsx2rr964z1gjl-linux-6.15-dev/lib/modules/6.15.0/build'
       > make[2]: Entering directory '/build/source'
       >   CC [M]  v4l2loopback.o
       > v4l2loopback.c: In function 'buffer_written':
       > v4l2loopback.c:1685:9: error: implicit declaration of function 'del_timer_sync' []
       >  1685 |         del_timer_sync(&dev->sustain_timer);
       >       |         ^~~~~~~~~~~~~~
       > make[4]: *** [/nix/store/gl4zb4minm4ddaz31lhsx2rr964z1gjl-linux-6.15-dev/lib/modules/6.15.0/source/scripts/Makefile.build:203: v4l2loopback.o] Error 1
       > make[3]: *** [/nix/store/gl4zb4minm4ddaz31lhsx2rr964z1gjl-linux-6.15-dev/lib/modules/6.15.0/source/Makefile:2003: .] Error 2
       > make[2]: *** [/nix/store/gl4zb4minm4ddaz31lhsx2rr964z1gjl-linux-6.15-dev/lib/modules/6.15.0/source/Makefile:248: __sub-make] Error 2
       > make[2]: Leaving directory '/build/source'
       > make[1]: *** [/nix/store/gl4zb4minm4ddaz31lhsx2rr964z1gjl-linux-6.15-dev/lib/modules/6.15.0/source/Makefile:248: __sub-make] Error 2
       > make[1]: Leaving directory '/nix/store/gl4zb4minm4ddaz31lhsx2rr964z1gjl-linux-6.15-dev/lib/modules/6.15.0/build'
       > make: *** [Makefile:53: v4l2loopback.ko] Error 2
       For full logs, run:
         nix log /nix/store/0g4mjwl3kg1iwmvjm410xxk9ckcligkn-v4l2loopback-0.13.2-6.15.drv
error: 1 dependencies of derivation '/nix/store/4cri51w2qb827kjfvrd28l81j2kimwjr-linux-6.15-modules.drv' failed to build
```

### Additional context

Seems like it's an upstream issue: https://github.com/v4l2loopback/v4l2loopback/pull/626. 

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.14.8-cachyos, NixOS, 25.11 (Xantusia), 25.11.20250525.62b852f`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.3`
 - nixpkgs: `/nix/store/nfqw3h3qc2mq37v6amw80b1rpi0g9x61-source`

### Notify maintainers

@moni

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/os-specific/linux/v4l2loopback/default.nix
+++ b/pkgs/os-specific/linux/v4l2loopback/default.nix
@@ -8,7 +8,7 @@
 }:
 
 let
-  version = "0.13.2";
+  version = "0.15.0";
 
 in
 stdenv.mkDerivation {
@@ -19,7 +19,7 @@ stdenv.mkDerivation {
     owner = "umlaeute";
     repo = "v4l2loopback";
     rev = "v${version}";
-    hash = "sha256-rcwgOXnhRPTmNKUppupfe/2qNUBDUqVb3TeDbrP5pnU=";
+    hash = "sha256-fa3f8GDoQTkPppAysrkA7kHuU5z2P2pqI8dKhuKYh04=";
   };
 
   hardeningDisable = [

